src/SketchPad.jsx

```diff
@@ -120,6 +153,9 @@ export default class SketchPad extends Component {
         onMouseMove={this.onMouseMove}
         onMouseOut={this.onMouseUp}
         onMouseUp={this.onMouseUp}
+        onTouchStart={this.onTouchStart}
```

Comment:
can we just pass down mouse event? like `onTouchStart={this.onMouseDown}` we don't need to create separate logic then in constructor